src/main/scala/io/viash/config/dependencies/GithubRepositoryTrait.scala

```diff
@@ -42,7 +42,7 @@ trait GithubRepositoryTrait extends AbstractGitRepository {
   def getCacheIdentifier(): Option[String] =
     Some(s"github-${repo.replace("/", "-")}${tag.map(_.prepended('-')).getOrElse("")}")
 
-  lazy val uri = s"https://github.com/$repo.git"
+  val uri = s"https://github.com/$repo.git"
   lazy val uri_ssh = s"git@github.com:$repo.git"
   val fakeCredentials = "nouser:nopass@" // obfuscate the credentials a bit so we don't trigger GitGuardian
   lazy val uri_nouser = s"https://${fakeCredentials}github.com/$repo.git"
```

Comment:
why make uri lazy but not uri_ssh or uri_nouser?

Reply:
`uri` used to be lazy, like `uri_ssh` and `uri_nouser`, which I believe is somewhat nicer, but since `uri` is used in a case class, it can't be lazy there. Scala 2 allowed that, Scala 3 doesn't.